Dockerfile

```diff
@@ -1,10 +1,18 @@
-FROM eclipse-temurin:21.0.3_9-jre
+FROM eclipse-temurin:21.0.4_7-jre
+
+ENV PATH_TO_JAR=/opt/eno-ws/eno-ws.jar
 WORKDIR /opt/eno-ws/
-COPY ./eno-ws/build/libs/*.jar /opt/eno-ws/eno-ws.jar
+COPY ./eno-ws/build/libs/*.jar $PATH_TO_JAR
 EXPOSE 8080
 
 RUN addgroup eno
 RUN useradd -g eno eno
 USER eno
 
-ENTRYPOINT ["java", "-jar",  "/opt/eno-ws/eno-ws.jar"]
+ENV JAVA_TOOL_OPTIONS_DEFAULT \
+    -XX:MaxRAMPercentage=75 \
+    -XX:+UseParallelGC
```

Comment:
Consider usage of ZGC which is better in many use cases : https://docs.oracle.com/en/java/javase/21/gctuning/z-garbage-collector.html#GUID-8637B158-4F35-4E2D-8E7B-9DAEF15BB3CD

Reply:
If you have memory problems, consider options to produce heap dumps : -XX:+HeapDumpOnOutOfMemoryError and -XX:HeapDumpPath=path